pyjob/sge.py

```diff
@@ -65,6 +66,22 @@ def info(self):
                     data[kv[0]] = kv[1]
         return data
 
+    @property
```

Comment:
given that we are using python3.7, we could (should?) make use of [`cached_property`](https://docs.python.org/3/library/functools.html#functools.cached_property) to avoid re-computation

Reply:
Unfortunately CCP4 probably won't move to python3 until the second half of the year, both @hlasimpk and will be needing python2 compatibility until then, which is a shame. As an alternative I can refactor this into `__init__()` so that it only gets computed once.

Reply:
I believe you resolved this with your change to staticmethod